native/cocos/renderer/gfx-vulkan/VKCommandBuffer.cpp

```diff
@@ -64,7 +64,7 @@ void CCVKCommandBuffer::doInit(const CommandBufferInfo & /*info*/) {
 }
 
 void CCVKCommandBuffer::doDestroy() {
-    CC_SAFE_DELETE(_gpuCommandBuffer);
+    _gpuCommandBuffer = nullptr;
```

Comment:
IntrusivePtr also has `reset()`.

Reply:
I took a look at the implementation of `unique_ptr`.  It overloads `operator=(nullptr_t)`. I can modify to the same style.
```cpp
unique_ptr& operator=(nullptr_t) noexcept {
    reset();
    return *this;
}
```

Reply:
What do you want to modify?

Reply:
I prefer to replace `reset()` with nullptr assginment. `reset` will be a little confused if we don't know it is a smart_ptr or the class has a `reset` member function.

Reply:
Got it.